Service/Model.Implementation/TodoController.cs

```diff
@@ -90,6 +115,12 @@ public Todo Create(string title)
         public void ChangeOrder(int id, int order)
         {
             var todo = Repository.Get(id);
+            if (todo == null)
+            {
+                throw new FaultException<ItemNotFoundFault>(
```

Comment:
Simple inheritance is impossible. A typical service contract has a view like
```
[OperationContract]
[FaultContract(typeof(ItemNotFoundFault))]
[FaultContract(typeof(DataValidationFault))]
void ChangeText(int id, string title, string desc);
```
WCF-server transports in the SOAP envelope only *Fault DTO object and then WCF-client rethrows a new FaultException exception with the received data. If I will make ItemNotFoundFault:FaultException then WCF-server will transport only empty object. I may make inheritance like
```
ItemNotFoundFaultException : FaultException<ItemNotFoundFault>
```
but it is a redundant thing.